src/docs/asciidoc/http-server.adoc

```diff
@@ -685,3 +685,108 @@ HTTP/2 200
 hello
 ----
 ====
+
+== Metrics
+HTTP server supports built-in integration with https://micrometer.io/[`Micrometer`].
+It exposes all metrics with prefix `reactor.netty.http.server`.
```

Comment:
Change it to be: It exposes all metrics with a prefix of `reactor.netty.http.server`.